.github/workflows/phpcs_on_pull_request.yml

```diff
@@ -3,14 +3,19 @@ name: Inspections
 jobs:
   runPHPCSInspection:
     name: Run PHPCS inspection
-    runs-on: ubuntu-latest
+    # Change below line to: `runs-on: ubuntu-latest` if not using self-hosted runners or if it is a public repo.
+    runs-on: self-hosted
```

Comment:
@chandrapatel should PHPCS checks run on this repo as well?

In that case this will have to be `runs-on: ubuntu-latest`. Because it is not recommended to use self-hosted runners on public repos and we follow that.

Reply:
@mrrobot47 Yes, phpcs checks should run on this repo.

Reply:
@mrrobot47 In that case, don't use self hosted.